### ISSUE TITLE
Quasar CLI - Capacitor v3 support

### ISSUE THREAD

**Issue by bazuka5801** [quasarframework/quasar#8366]
**Describe the bug**
After upgrade I have bug on IOS with code inside quasar install script.

**Screenshots**
![image](https://user-images.githubusercontent.com/11452353/105580231-26abb280-5d9c-11eb-8f1e-6e8bdda2dfe9.png)
![image](https://user-images.githubusercontent.com/11452353/105580293-928e1b00-5d9c-11eb-90d5-37ced820961c.png)

**Platform (please complete the following information):**
OS: Mac OS BigSur
Node: 12+
Yarn: Latest
Browsers: Chrome
iOS: 12.4

**Comment by bazuka5801** [comment]
After install App plugin as npm package, I got next error
![image](https://user-images.githubusercontent.com/11452353/105601282-17c9fd80-5da6-11eb-9b52-1f7459d345d4.png)

**Comment by bazuka5801** [comment]
@hawkeye64 Next news!
I fixed it, problem with ordering initialization quasar and capacitor.
Steps to fix:
1) change in capacitor.config.js -> "bundledWebRuntime" to "true"
2) add in src/index.template.html -> "<script src="capacitor.js"></script>" before `.q-app` div
3) npx cap sync in src-capacitor
4) build app (quasar build -m capacitor -T [android|ios]

**Comment by bazuka5801** [comment]
But this is not good solution, I was trying find root of reordering, maximum i found quasar install js, where History.instal(...), but we must load capacitor before run global install.
How we can initialize capacitor?

**Comment by bazuka5801** [comment]
New research, then it throw new error
![image](https://user-images.githubusercontent.com/11452353/107652249-6ece3e80-6c91-11eb-93e2-ce077abc73f6.png)
Need to install Splash Screen plugin
```bash
cd src-capacitor
yarn add @capacitor/splash-screen
npx cap sync
```

**Comment by slavabelb** [comment]
@bazuka5801 Could you describe pls you updated capacitor? I tried to follow https://capacitorjs.com/docs/v3/updating/plugins/3-0

**Comment by slavabelb** [comment]
> @bazuka5801 Could you describe pls how did you update capacitor? I tried to follow https://capacitorjs.com/docs/v3/updating/plugins/3-0

**Comment by bazuka5801** [comment]
You need:
1. Enable `bundledWebRuntime` to `true` in **capacitor.config**
2. Install all default plugins you need, must for quasar [App, SplashScreen] (installation process in [docs](https://capacitorjs.com/docs/v3/apis) v3)
3. Try it!
If you have any issue put it down, I will try to fix it!

For quasar update I think:
1) We need add checks for default plugins, because in default it was not installed

I following some problem which I don't resolve, when updating I need enable bundled web runtime in header of html, and beta.2 version not worked for me :(
But I interested in solving problem for quasar and my company projects

**Comment by rstoenescu** [comment]
Capacitor v3 support will come with "@quasar/app" v3.0.0-beta.8 + Quasar v2.0.0-beta.8.
Will backport for Quasar v1 too.

Note for myself: docs still need update.

**Comment by rstoenescu** [comment]
HTTPS not working (only HTTP). Need to investigate and talk with the Capacitor team.

**Comment by bazuka5801** [comment]
@rstoenescu
On both platforms  HTTPS not working?

**Comment by rstoenescu** [comment]
Yes. Not working because the Capacitor team changed a lot under the covers, so the old hooks that Quasar was using are no longer valid. Will need the Capacitor team involved again. However, we'll ship the non-HTTPS support in the next Quasar v2 beta (and also backport to Quasar v1). But until we're in sync with the Capacitor team on the HTTPS thing, we'll still spawn a Capacitor v2 project in /src-capacitor. However anyone will be able to use Capacitor v3 as well (by just upgrading the Capacitor deps in /src-capacitor) --- just not with HTTPS.

**Comment by bazuka5801** [comment]
@rstoenescu Okay, I don't understand about HTTPS, what did you mean 'HTTPS':
1) Requests from client (capacitor) - to HTTPS server
2) HTTPS dev server which host dev app

Or both these not working?

**Comment by rstoenescu** [comment]
Talking about HTTPS dev server.

**Comment by bazuka5801** [comment]
> and also backport to Quasar v1

@rstoenescu We are looking forward to the v1 backport  💪

**Comment by bazuka5801** [comment]
@rstoenescu 
When do we expect the backport?

**Comment by rstoenescu** [comment]
As time allows, in a fee days.

**Comment by bazuka5801** [comment]
Up

**Comment by bazuka5801** [comment]
@rstoenescu Any updates on v1? We need this

**Comment by Sommmmy** [comment]
any news on when is the backport coming? needing it too @rstoenescu

**Comment by bazuka5801** [comment]
@rstoenescu Added PR!

**Comment by rstoenescu** [comment]
@bazuka5801 thanks for backporting it to Q v1!

Will be available with Quasar v1.15.5+ and "@quasar/app" v2.2.0+.
